src/static/stylus/base_template.styl

```diff
@@ -5,20 +5,54 @@
  */
 #bg
     z-index -1
-    background rgba(22, 12, 160, 0.9)
     position absolute
     top 0
     bottom 0
     left 0
     right 0
+    background rgba(19, 48, 70, 0.9)
```

Comment:
same with this color, if we use it often

Reply:
Added Color Reference to base_template.styl